api/src/main/java/io/lumeer/engine/api/LumeerConst.java

```diff
@@ -165,4 +171,28 @@ public static LinkDirection fromString(final String s) {
       public static final int WRITE = 2;
       public static final int EXECUTE = 1;
    }
+
+   public static class Project {
+
+      public static final String COLLECTION_NAME = "_system-project";
+      public static final String ATTR_PROJECT_ID = "project-id";
+      public static final String ATTR_ORGANIZATION_ID = "organization-id";
+      public static final String ATTR_PROJECT_NAME = "project-name";
+      public static final String ATTR_USERS = "users";
+      public static final String ATTR_USER = "user";
```

Comment:
`Users` is name of the array field and `user` (annd also `user_roles` ) is the name of the field inside documents stored in this users array.